app/models/furniture_placement.rb

```diff
@@ -42,4 +42,10 @@ def form_template
   def configurable?
     furniture.form_template != "furniture_placements/noop"
   end
+
+  def write_attribute(name, value)
+    super
+  rescue => e
```

Comment:
Hrm. Would it be possible to rescue a more specific error? ("missing attribute" or whatever?)

Reply:
Yea, I was... YOLOing.